src/lib/fauna.mjs

```diff
@@ -73,7 +77,7 @@ export const runQuery = async ({
 }) => {
   // Check for required arguments.
   if (!query) {
-    throw new Error("A query is required.");
+    throw new CommandError("A query is required.");
```

Comment:
is this not validation? the line's fuzzy, but it seems to me that validation errors are sort of all errors we can determine strictly from argv without touching fs, network, etc.

Reply:
+1, because hideHelp is false on ValidationError

Reply:
I can switch it. The problem is that this is library code and we should never really get this far without a query. You can't write a helpful validation message (i.e. use `[fql] or --input`) in this context.

Reply:
trying a few things like leaving the query arg missing or providing an empty string catch the error before this point. So this may be moot? If the other code is working query should always be truthy by this point.